src/embind/embind.js

```diff
@@ -584,6 +589,46 @@ var LibraryEmbind = {
     });
   },
 
+#if WASM_BIGINT
+  _embind_register_bigint__deps: [
+    'embind_repr', '$readLatin1String', '$registerType', '$integerReadValueFromPointer'],
+  _embind_register_bigint: function(primitiveType, name, size, minRange, maxRange) {
+    name = readLatin1String(name);
+
+    var shift = getShiftFromSize(size);
+
+    var isUnsignedType = (name.indexOf('u') != -1);
+
+    // maxRange comes through as -1 for uint64_t (see issue 13902). Work around that temporarily
+    if (isUnsignedType) {
+      maxRange = 18446744073709551615n;
```

Comment:
BigInt literals like this cause issues currently (see failing tests). Sadly writing `BigInt(..)` won't work here as double rounding will occur, but how about this:
```suggestion
      maxRange = (BigInt(1) << BigInt(64)) - BigInt(1);
```

Reply:
Suggestion gratefully accepted.